.travis.yml

```diff
@@ -14,9 +14,11 @@ before_script:
 script:
   - pylama pontoon
   - python manage.py collectstatic --noinput
-  - python manage.py test --with-coverage
   - ./node_modules/.bin/webpack
+  - npm test
+  - python manage.py test --with-coverage
   - py.test --cov-append --cov-report=term --cov=. -v
+  - (./node_modules/.bin/eslint .; exit 0)
```

Comment:
I like that, but can you add a comment to notice that it should only be temporary?

Reply:
for sure